js/util/script_detection.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const isVerticalWritingModeRegExp = new RegExp([
+    '[ᄀ-ᇿ]',
+    '[가-힣]',
+    '[ㄱ-ㆎ]',
+    '[一-鿌]',
+    '[㐀-䶵]',
+    // eslint-disable-next-line no-irregular-whitespace
+    '[　-〿]',
```

Comment:
@1ec5 Are you sure? I'm seeing an ideographic space here. Would you prefer the `\u` version in this case?